### PR TITLE
Premium file bandwidth limits

### PR DESCRIPTION
**Pull request by rickle-msft**

### CODE REVIEW

File: sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/implementation/AzureFileStorageImplBuilder.java [hunk 1]
```diff
@@ -6,16 +6,20 @@
 
 import com.azure.core.annotation.ServiceClientBuilder;
 import com.azure.core.http.HttpClient;
+import com.azure.core.http.HttpHeaders;
```
Comment (seanmcc-msft) [inline sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/implementation/AzureFileStorageImplBuilder.java:9]: What are the changes in this file for?  They seems unrelated to premium file bandwidth limits.

Comment (rickle-msft) [reply sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/implementation/AzureFileStorageImplBuilder.java:9]: It's a generated file, so I think these must be changes that came out of whatever updates were made to the generator.

**Review by seanmcc-msft — APPROVED** [review]
